tests/basics/python34.py

```diff
@@ -6,27 +6,23 @@
     print("SKIP")
     raise SystemExit
 
-# from basics/fun_kwvarargs.py
-# test evaluation order of arguments (in 3.4 it's backwards, 3.5 it's fixed)
-def f4(*vargs, **kwargs):
-    print(vargs, kwargs)
+
 def print_ret(x):
     print(x)
     return x
-f4(*print_ret(['a', 'b']), kw_arg=print_ret(None))
```

Comment:
I think this test should be retained, it's a good test for evaluation order.  Maybe put it back in `basics/fun_kwvarargs.py` now that both uPy and CPy agree on the output?

Reply:
done.